components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -606,15 +606,30 @@ class NewTabPage extends React.Component<Props, State> {
     })
   }
 
+  updateGeminiAssetAddress = (asset: string, address: string) => {
+    this.props.actions.setGeminiAssetAddress(asset, address)
+    void generateQRData(address, asset, this.setGeminiAssetDepositQRCodeSrc)
```

Comment:
I know this is moving existing code but I'm wondering why we would need the void keyword here?

Reply:
W/o `void`, previous lint complained like need to handle promise.
But, our new lint doesn't complains anymore. deleted.